src/krati_Jain/Assignment8NumbersWhile.java

```diff
@@ -0,0 +1,65 @@
+package krati_Jain;
+
+class Assignment8NumbersWhile {
+	
+	void evenNumbers(int startNum1, int endNum1) {
+		int totalCount = 1;
```

Comment:
you can handle without this totalCount variable.

refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files